src/rp/util/string.clj

```diff
@@ -28,3 +28,8 @@
   (some-> s
           parse-double*
           double))
+
+(defn non-blank-string?
+  "Returns true if x is a non-blank string"
+  [x] (and (string? x)
```

Comment:
There's also things like this in Clojure core: https://github.com/clojure/clojure/blob/e547d35bb796051bb2cbf07bbca6ee67c5bc022f/src/clj/clojure/core.clj#L1475-L1479

I'd say we stick with line-break across the board to keep things internally consistent.